lib/lnd_client.ex

```diff
@@ -118,6 +119,19 @@ defmodule LndClient do
     GenServer.call(__MODULE__, { :add_invoice, invoice })
   end
 
+  @doc """
+  Takes a %Lnrpc.SendRequest and attempts to pay the invoice
+
+  ## Examples
+
+      iex> Lnrpc.SendRequest.new(payment_request: "lnbcrt500paymentrequest") |> LndClient.send_request_sync
```

Comment:
Seems like `LndClient.send_request_sync` should be `LndClient.send_payment_sync` here

Reply:
Ah yes. Good catch!